Makefile

```diff
@@ -59,6 +59,10 @@ build/pyodide.asm.js: \
 		let clearImmediate = globalThis.clearImmediate;\
 		let baseName, fpcGOT, dyncallGOT, fpVal, dcVal;\
 	' build/pyodide.asm.js
+	# Remove last 6 lines of pyodide.asm.js, see issue #2282
+	for _ in {1..6} ; do \
+		sed -i '$$d' build/pyodide.asm.js ; \
```

Comment:
Might be better to pipe to avoid overwriting the same file 6 times? https://stackoverflow.com/a/24298204

The line position doesn't sound very robust.
Maybe add a TODO that this should be removed after an upstream fix?

Otherwise LGTM.

Reply:
Maybe I can switch to one of the other approaches listed there.